app/code/core/Mage/Catalog/Model/Resource/Category.php

```diff
@@ -728,6 +728,7 @@ public function getChildren($category, $recursive = true)
                 array()
             )
             ->where($checkSql . ' = :scope');
+        $select->order("m.position ASC");
```

Comment:
Can we include this as part of the fluent call right after the `where` and drop the `$select`?

Reply:
Thanks for your support. I have refactored it.